Sources/SnowManLogic/GameLogic/Game.swift

```diff
@@ -0,0 +1,45 @@
+public class Game {
+	var word: String
+	var guessManager: GuessManager
+	var view: View
+	var guess: String
+
+	public init(word: String, guessManager: GuessManager, view: View) {
+		self.word = word
+		self.guessManager = guessManager
+		self.view = view
+		guess = ""
+	}
+
+	public func playGame() {
+		while (gameIsOver() == false) {
+			playerTurn()
+		}
+		view.displayResults(word, winner: isWinner())
```

Comment:
passing a boolean as a flag is kind of a code smell, the conditional should take place here.